src/utilities/getComputedStyleValue.ts

```diff
@@ -0,0 +1,22 @@
+// https://stackoverflow.com/a/62557418/3511012
```

Comment:
If we're introducing generic utilities in this repository can we keep the naming structure we've used in our other projects? `utilities/styles.ts` for example.